spec/git_helper_spec.rb

```diff
@@ -145,37 +145,37 @@
     end
 
     it 'checks if a tag and a branch point to the same commit' do
-      same_commit = described_class.point_to_same_commit?('1.0', 'another-branch')
+      same_commit = described_class.point_to_same_commit?('1.0', 'another-branch', remote_name: nil)
```

Comment:
Added a test using `remote_name: 'origin'`,

```ruby
    it 'checks if commits between the same branch point to the same commit' do
      same_commit = described_class.point_to_same_commit?('feature-branch', 'feature-branch', remote_name: 'origin')
      expect(same_commit).to be true
    end
```

And got this failure:

```
  1) FastlaneCore::Helper::GitHelper point_to_same_commit?(ref1, ref2) checks if commits between the same branch point to the same commit
     Failure/Error: ref1_commit.sha == ref2_commit.sha

     Git::FailedError:
       git '--git-dir=/private/var/folders/dq/cdqxvx3s5ps75564rpmb_dc00000gn/T/d20240906-32427-fzffvl/.git' '--work-tree=/private/var/folders/dq/cdqxvx3s5ps75564rpmb_dc00000gn/T/d20240906-32427-fzffvl' '-c' 'core.quotePath=true' '-c' 'color.ui=false' 'rev-parse' 'origin/feature-branch'  2>&1
       status: pid 32722 exit 128
       output: "fatal: ambiguous argument 'origin/feature-branch': unknown revision or path not in the working tree.\nUse '--' to separate paths from revisions, like this:\n'git <command> [<revision>...] -- [<file>...]'\norigin/feature-branch\n"
```

I think that's expected because the local branches used in this test do not have a remote. Am I right?

What do you think of adding a note about this, e.g. at the start of the `context` (why not `describe`?) for this method.

```ruby
# We cannot test the happy using a remote name because the repo we use for the tests does not have a remote.
let(:remote_name) { nil }
```

Reply:
Great suggestions, thanks! Updated on 36cd7bc710a9e01b6003b3a963ff643cb804a87c.